config/istio/envoyfilter.yaml

```diff
@@ -19,7 +19,7 @@ spec:
               subFilter:
                 name: 'envoy.filters.http.router'
       patch:
-        operation: INSERT_BEFORE
+        operation: INSERT_FIRST
```

Comment:
there's another file when deploying the broker on your host rather than in cluster that needs updating too in config/dev/envoyfilter.yaml